src/iter/test.rs

```diff
@@ -2104,4 +2104,4 @@ fn check_update() {
     v.par_iter_mut().update(|v| v.push(0)).for_each(|_| ());
 
     assert_eq!(v, vec![vec![1, 0], vec![3, 2, 1, 0]]);
-}
+}
```

Comment:
I don't know why your editor removed the final newline, but let's revert that please.

Reply:
Hmm, it doesn't appear to show in the inline review, but the full github diff displays this change with "No newline at end of file", which is what I want to revert.